go/cmd/vtctldclient/cli/json.go

```diff
@@ -25,6 +25,11 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+const (
```

Comment:
two things:

1. do these need to be exported?
2. maybe for naming `JSONIndent` or `JsonIndent`? i'm not sure what the Go convention says here: https://google.github.io/styleguide/go/decisions#initialisms

Reply:
I can just remove them if you prefer? I was initially going to use them in the vdiff cmd package but then switched to just using the cli MarshalPretty method.